code/game/objects/items/storage/bags.dm

```diff
@@ -108,6 +108,7 @@
 
 /obj/item/storage/bag/ore/ComponentInitialize()
 	. = ..()
+	AddComponent(/datum/component/rad_insulation, 0.01) //please datum mats no more cancer

```

Comment:
I dont think you need this anymore?

Reply:
we do else kminers still get bad rad poisoning